code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -102,8 +102,10 @@
 	name = "adminordrazine pill"
 	desc = "It's magic. We don't have to explain it."
 	icon_state = "pill16"
-	volume = 50
-	list_reagents = list(/datum/reagent/medicine/adminordrazine = 50)
+	volume = 5
```

Comment:
why is this changed?

Reply:
because it sometimes appears in the medbay rooms and i didn't want people getting 50u of adminordrazine when as far as i can tell, 1u will do the full rejuvinate